src/rulesets/oas2/functions/oas2UnusedDefinition.ts

```diff
@@ -0,0 +1,66 @@
+import { JsonPath } from '@stoplight/types';
+import { IFunction, IFunctionResult } from '../../../types';
+
+function* refPointerIterator(
+  obj: object,
+  path: JsonPath = [],
+  alreadyYielded: string[] = [],
+): IterableIterator<[JsonPath, string]> {
```

Comment:
Seems like you don't make use of path, so let's get rid of it :)
```suggestion
): IterableIterator<string> {
```

Reply:
Fixed

Reply:
This has eventually been dropped.